components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/edit-finish-ajaxprocessor.jsp

```diff
@@ -84,6 +87,9 @@
         app.setOAuthVersion(oauthVersion);
         app.setPkceMandatory(pkceMandatory);
         app.setPkceSupportPlain(pkceSupportPlain);
+        app.setUserAccessTokenExpiryTime(Long.parseLong(userAccessTokenExpiryTime));
```

Comment:
The input type is text, so it is perfectly possible for someone to type a string instead of a number. Shouldn't we do the validation either here or enforce it at UI?

Reply:
or we should handle that exception and indicate a clear error message. But IMO better to validate in UI itself.

Reply:
UI validation is done. So this is not necessory